maxSpeed : 0.3
minSpeed : -0.15
maxYaw_rate : 140 #[degree/s]
maxAccel : 0.5
maxYawVel_rate : 120 #[degree/ss]
v_resolution : 0.01
w_resolution : 0.5 #[degree/s]
dt : 0.1
predictTime : 2
angleCost_gain : 0.35
distanceCost_gain : 0.2
obstacleCost_gain : 1.6
robot_stuck_value : 0.001

map_frame_id : map
base_link_frame_id : base_footprint
loop_rate : 10